wiki/Game_modifier/Flashlight/ko.md

```diff
@@ -0,0 +1,80 @@
+---
+stub: true
+태그:
```

Comment:
```suggestion
tags:
```
These **must not** be translated. The tags in itself can be translated though. See [here](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#tagging-articles) for more information.